<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CumulateDataPage</class>
 <widget class="QWidget" name="CumulateDataPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);
}

QLabel
{
background-color:rgb(51,153,255);
border:0px solid white;
font:20px &quot;SimHei&quot;;
color:white;
}

QPushButton
{
outline:none;
font:25px, &quot;SimHei&quot;;
}</string>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>牵引能耗</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>75</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>再生制动能耗</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>145</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>运行总里程</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>辅助能耗</string>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>215</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>TCMS总运行时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>检修里程</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>285</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>空压机2运行时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>空压机1运行时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTractionEnergy">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReEngy">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>75</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAxPower">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTotalKilo">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>145</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnServiceKilo">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTcmsTime">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>215</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAirPressTime1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAirPressTime2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>285</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAxPowerTimeY">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTcmsTimeTimeY">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>215</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTractionEnergyTimeY">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReEngyTimeY">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>75</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTotalKiloTimeY">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>145</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnServiceKiloTimeY">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAirPressTime1TimeY">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAirPressTime2TimeY">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>285</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>85</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>起始时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>91</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>累计数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8_4">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>40</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTractionEnergyTimeM">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_5">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>40</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTractionEnergyTimeD">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_6">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>40</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8_7">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>75</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8_8">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>75</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReEngyTimeM">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>75</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_9">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>75</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReEngyTimeD">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>75</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_10">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>110</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8_11">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>110</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAxPowerTimeD">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_12">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>110</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAxPowerTimeM">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_13">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>145</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8_14">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>145</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTotalKiloTimeD">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>145</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_15">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>145</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTotalKiloTimeM">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>145</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_16">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>180</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnServiceKiloTimeM">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_17">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>180</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8_18">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>180</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnServiceKiloTimeD">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_19">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>215</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTcmsTimeTimeM">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>215</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_20">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>215</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8_21">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>215</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTcmsTimeTimeD">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>215</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_22">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>250</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAirPressTime1TimeM">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_23">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>250</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8_24">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>250</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAirPressTime1TimeD">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_25">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>285</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAirPressTime2TimeM">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>285</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8_26">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>285</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8_27">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>285</y>
     <width>21</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAirPressTime2TimeD">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>285</y>
     <width>31</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>681</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
font:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
